repl/convert_ast.py

```
@@ -7,19 +7,68 @@
 import qwerty_ast
 #################### COMMON CODE FOR BOTH @QPU AND @CLASSICAL DSLs ####################
 
+# quick fix, going to create another file most likely for errors
+class QwertySyntaxError(Exception):
+    pass
+
 def convert_to_qwerty(py_ast):
     print(ast.dump(py_ast, indent=4))
-    value = unpack_ast(py_ast)
-    if value is None:
-        return "Invalid Expression: Not a qubit literal or Tensor"
-    tensor = qwerty_ast.NodeBox.new_qubit_tensor(py_ast.body[0].value.value)
-    return tensor.get_tensor()
 
-def unpack_ast(ast):
+    if not isinstance(py_ast, (ast.Interactive, ast.Module)):
+        return "Invalid AST"
+    
+    if (len(py_ast.body) == 1):
+        value = py_ast.body[0]
+    else:
+        return "Invalid body; length of body is not 1"
```

This should be `raise QwertySyntaxError("Invalid body; length of body is not 1")`, but I think we can say something more useful too.

The only case that comes to mind of this happening would be someone saying, for example, `'0'; '1'`. Here's how I verified this in the Python interpreter (`python3` on the command line):
```python
>>> import ast
>>> print(ast.dump(ast.parse("'0'; '1'", mode='single'),indent=4))
Interactive(
    body=[
        Expr(
            value=Constant(value='0')),
        Expr(
            value=Constant(value='1'))])
```

I figure this could happen if some sophomore is using this and accidentally typed `;` instead of something else. What is most helpful for that student would be to say something like "Multiple statements are not supported. Please do not write semicolons `;`." (But I support any other ideas you have for phrasing)